css/main.css

```
@@ -8,3 +8,47 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+button {
+  width: 100px;
+  height: 15px;
+  font-family: Poppins;
+  font-size: 10px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
```

Não é necessário, já é assim por padrão. 
Pode pagar

---

Se eu tirar a 
```suggestion
  font-size: 10px;
``` a letra vai ficar muito grande